mne_bids/mne_bids.py

```diff
@@ -655,10 +656,20 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
                          % (''.join(ALLOWED_EXTENSIONS), ext))
 
     # Copy the imaging data files
-    # Re-save FIF files to fix the file pointer for files with multiple parts
-    # This is WIP, see: https://github.com/mne-tools/mne-python/pull/5470
     if ext in ['.fif']:
-        raw.save(raw_file_bids, overwrite=overwrite)
+        n_rawfiles = len(read_raw_fif(raw_fname).filenames)
+        if n_rawfiles > 1:
+            # TODO Update MNE requirement to version 0.17 when it's released
+            if check_version('mne', '0.17.dev'):
+                split_naming = 'bids'
+            else:
+                raise NotImplementedError(
+                'Renaming split fif files is not supported on your version '
+                'of MNE. Please upgrade to at least "0.17.dev". '
+                'Please contact MNE developers if you have any questions.')
```

Comment:
didn't quite work @teonbrooks . See the error on circleci.

Reply:
the commit below should fix it

Reply:
can you add a test for this? thanks

Reply:
the test for the split files live in mne-python. would this just be a test of the `check_version` call?

Reply:
yeah basically to cover `L666-669` ...